library/core/src/lib.rs

```diff
@@ -96,6 +96,8 @@
 #![allow(explicit_outlives_requirements)]
 #![allow(incomplete_features)]
 #![warn(multiple_supertrait_upcastable)]
+// Do not check link redundancy on bootstraping phase
+#![cfg_attr(not(bootstrap), allow(rustdoc::redundant_explicit_links))]
```

Comment:
Compiler seems does not allow crate attribute declares at `primitive_docs.rs` in both core and std?

Reply:
But after added that attribute at `lib.rs`, compiler still warns about the lint.

Reply:
I think allow it would work, after added `#![cfg_attr(not(bootstrap), allow(rustdoc::redundant_explicit_links))]`, the lint muted and stdlib documentation works fine.